docs-user/js/init.js

```diff
@@ -6,8 +6,8 @@ window.$docsify = {
   loadNavbar: true,
   subMaxLevel: 2,
   // If changing the Google Analytics ID, change it in res/before-load.js as well.
-  ga: 'UA-35433268-81',
+  // ga: 'UA-35433268-81',
```

Comment:
Do we need to keep this around still?

Reply:
I was thinking that we might want to see our old value easily, but I see I can find it easily on the google analytics website anyway, so I guess I can remove it here.